lms/djangoapps/ora_staff_grader/urls.py

```diff
@@ -15,16 +15,19 @@
 
 urlpatterns += [
     url('mock/', include('lms.djangoapps.ora_staff_grader.mock.urls')),
-    path(
```

Comment:
I think path is preferred over URL, I'd go the other way.

Reply:
When I tried using path, things broke... 🙁

Reply:
Ah, figured out why. Updating